test-requirements.txt

```diff
@@ -1,2 +1,4 @@
-pytest==5.2.1
+pytest>=5.2.1
+mock>=3.0.5
```

Comment:
I don't know about this library, but `unttest.mock` is a part of the standard library.